lib/CL/pocl_llvm_api.cc

```diff
@@ -854,10 +1022,18 @@ int pocl_llvm_get_kernel_metadata(cl_program program,
                               content.str().size());
 #endif
 
+#if defined(LLVM_OLDER_THAN_3_9)
```

Comment:
Just define the whole pocl_llvm_get_kernel_metadata inside LLVM_OLDER_THAN_3_9 selection, no need to have the indirection here as both are not needed at the same time.